src/nrepl/project-types.ts

```diff
@@ -550,12 +550,12 @@ async function cljCommandLine(
             parsed && parsed.aliases != undefined
                 ? Object.keys(parsed.aliases)
                 : [];
-        for (const a in projectAliases) {
-            const aliasKey = unKeywordize(projectAliases[a]);
+        for (const projectAlias of projectAliases) {
```

Comment:
Using for/in is typically not what people want when they're iterating over an array and has some really weird and unexpected behaviors. for/of is generally better in every respect.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-for-in-array.md